TelepresenceBot/core/src/main/java/com/novoda/tpbot/support/Observable.java

```diff
@@ -15,7 +13,7 @@ protected Observable() {
 
     public synchronized Observable<T> attach(Observer<T> observer) {
         if (observer == null) {
-            throw new DeveloperError("You cannot attach a null observer");
+            throw new NullPointerException("You cannot attach a null observer");
```

Comment:
I retracted my 👍  - it's already added in this project (so no difference in APK) just in the wrong module. Moving it to core would mean both modules can use it so "didn't have access" wouldn't be valid.